File: backend-old/TransportStatic/Models/VehicleBoarding.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace TransportStatic.Models;

[Table("vehicle_boardings")]
public class VehicleBoarding
{
    [Column("vehicle_category_id")]
    [Required]
    public string VehicleCategoryId { get; set; } = null!;
    
    [Column("child_sequence")]
    public string ChildSequence { get; set; } = null!;
    
    [Column("grandchild_sequence")]
    public int? GrandchildSequence { get; set; }
    
    [Column("boarding_area_id")]
    [Required]
    public string BoardingAreaId { get; set; } = null!;
}

File: backend/TransportRealtime/models/vehicle_boarding.go
package models

type VehicleBoarding struct {
	VehicleCategoryId  string `json:"vehicle_category_id"`
	ChildSequence      int    `json:"child_sequence"`
	GrandchildSequence int    `json:"grandchild_sequence"`
	BoardingAreaId     string `json:"boarding_area_id"`
}
